mmdet/models/plugins/non_local.py

```diff
@@ -0,0 +1,82 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from mmcv.cnn import constant_init
+
+from ..utils import ConvModule
+
+
+class NonLocalBlock2D(nn.Module):
```

Comment:
To be consistent with PyTorch's naming conventions, this class may be renamed to `NonLocal2d`.